File: Who Likes This/whoLikesThis.js
function likes(names) {
    let like = names
    switch(like.length){
        case  0 : return "no one likes this";
        case  1 : return `${like[0]} likes this`;
        case 2 : return `${like[0]} and ${like[1]} like this`;
        case  3 : return `${like[0]}, ${like[1]} and ${like[2]} like this`;
        default : return `${like[0]}, ${like[1]} and ${like.length - 2} others like this`;
    }
  }

File: Who Likes This/whoLikesThis.py
def likes(names):
    likes = names
    if(len(likes) == 0):
        return "no one likes this"
    elif(len(likes) > 0):
        if(len(likes) == 1):
            return likes[0]+" likes this"
        if(len(likes) == 2):
            return likes[0]+" and "+likes[1]+" like this"
        if(len(likes) == 3):
            return likes[0]+", "+likes[1]+" and "+likes[2]+" like this"
        if(len(likes) > 3):
            return likes[0]+", "+likes[1]+" and "+str(len(likes)-2)+" others like this"
    pass